pints/_nested/__init__.py

```diff
@@ -0,0 +1,62 @@
+#
+# Sub-module containing MCMC inference routines
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import division
+import pints
+import numpy as np
+
+class NestedSampler(object):
+    """
+    Takes a :class:`LogLikelihood` function and returns a nested sampler.
+    
+    Arguments:
+    
+    ``function``
+        A :class:`LogLikelihood` function that evaluates points in the
+        parameter space.
+
+    """
+    def __init__(self, log_likelihood, aPrior):
```

Comment:
We're using camelcasing for class names only in Pints, and underscores in variable and method/function names (which is the more common way to do it for python libraries)